src/docs/patterns/internationalization.mdx

```diff
@@ -13,42 +16,83 @@ import Image4 from '../../app/assets/design-patterns/internationalization/web-in
+
+## Checking for Spacing
+
+Different languages/writing systems may require different vertical and horizontal space. To ensure flexibility, avoid using fixed dimensions for necessary UI components such as buttons, labels, and badges. Also, consider setting different default display font sizes for languages that may have smaller glyphs.
+
+> **NOTE:** PX Blue recommends using [Noto Sans](https://www.google.com/get/noto/#sans-lgc) for all non-western languages. You can read more on our [Typography](/style/typography) page.
+
+## Bi-directionality/Right-to-Left (RTL) Support
 
 <MaterialDesignDescription
     title={'Bidirectionality'}
     description={'Learn about mirroring layout, mirroring elements, and localization.'}
     url={'https://material.io/design/usability/bidirectionality.html'}
 />
 
-## Technical Implementation
+Some languages, such as Arabic and Hebrew, are read from right to left. Applications supporting these languages will need to mirror the UI so the natural flow is from right to left. For example, navigation drawers should be displayed on the right side of the screen, icons should be to the right of the text in buttons, and any icons with directionality should be mirrored.
+
+<ImageGrid
+    images={[UIFlippedImage]}
+    regularWidth
+    caption={[
+        '1: A PX Blue user interface in right-to-left language (English).',
+        '2: The same PX Blue user interface in left-to-right language (Arabic).',
+    ]}
+/>
+<ImageGrid
+    images={[IconsFlippedImage]}
+    regularWidth
+    caption={[
+        '1: Icons that indicate directions should be mirrored.',
+        '2: Icons with glyphs and symbols should never be mirrored.',
+    ]}
+/>
+
+## Curtural differences
+
+Sometimes certain UI elements convey different meaning in different cultures. For example, in America, the red color in the stock market indicates a decline, while in China the same color indicates a rise. A thumb-up gesture usually expresses an approval in English-speaking country, yet the gesture can have a negative meaning in Iraq. When designing for a new market, you are encouraged to have a local consultant run an audit for your application.
+
+<Divider />
+
+# Developers
 
 There are a number of available libraries to assist with implementing internationalization in your applications.
 
-### Angular
+## Angular
 
 For projects that need the ability to swap between languages, we recommend using [ngx-translate](https://github.com/ngx-translate/core). As an alternative, the Angular framework has a powerful built-in [i18n tool](https://angular.io/guide/i18n) available, but it requires a separate web app be built for each supported language (e.g. an English version, a Spanish version, etc). This is ideal for larger projects that don't require in-app language switching and don't support more than a few locales.
 
-### React or React Native
+## React and React Native
```

Comment:
Material-UI is not applicable to React Native.

Reply:
You'll want to mention the I18nManager from React Native (for dealing with RTL) and also i18next/react-i18next for translations.